projects/laji/src/app/+haseka/haseka.routes.ts

```diff
@@ -1,74 +1,48 @@
 import { RouterModule, Routes } from '@angular/router';
 import { ModuleWithProviders } from '@angular/core';
 import { HasekaComponent } from './haseka.component';
-import { HaSeKaTermsOfServiceComponent } from './terms-of-service/terms-of-service.component';
-import { HaSeKaFormComponent } from './form/haseka-form.component';
-import { DocumentDeActivateGuard } from '../shared/guards/document-de-activate.guard';
 import { OwnSubmissionsComponent } from './own-submissions/own-submissions.component';
 import { TemplatesComponent } from './templates/templates.component';
 import { OnlyLoggedIn } from '../shared/route/only-logged-in';
-import { HasekaFeedbackComponent } from './haseka-feedback/haseka-feedback.component';
 import { VihkoHomeComponent } from './vihko-home/vihko-home.component';
-import { TemplateHasekaFormComponent } from './template-haseka-form/template-haseka-form.component';
 
 export const hasekaRoutes: Routes = [
   {
     path: '',
-    // Feedback button is handled globally everywhere else but on Vihko forms, since the button shouldn't be shown on some forms.
-    component: HasekaFeedbackComponent,
+    component: HasekaComponent,
+    children: [
+      {path: '', pathMatch: 'full', redirectTo: 'home'},
+      {path: 'home', pathMatch: 'full', component: VihkoHomeComponent},
+      {path: 'forms', pathMatch: 'full', redirectTo: 'home'},
+      {path: 'ownSubmissions', pathMatch: 'full', canActivate: [OnlyLoggedIn], component: OwnSubmissionsComponent},
+      {path: 'templates', pathMatch: 'full', canActivate: [OnlyLoggedIn], component: TemplatesComponent},
+      {path: 'tools',  canActivate: [OnlyLoggedIn], loadChildren: () => import('./tools/tools.module').then(m => m.ToolsModule)},
+    ],
+  },
+  {
+    path: 'terms-of-service',
+    pathMatch: 'full',
+    redirectTo: '/about/5809'
```

Comment:
> The map settings differ between these two:
https://dev.laji.fi/project/MHL.65/form/MHL.65/places/new
https://211003.dev.laji.fi/project/MHL.65/form/MHL.65/places/new

Ah yes, I had removed [a code block that copies map options from the document form](https://github.com/luomus/laji/blob/development/projects/laji/src/app/%2Bproject-form/form/named-place/np-edit-form/np-edit-form.component.ts#L236-L243), since the named place geometry isn't the same thing as the document gatherings geometry and shouldn't be treated as such. I went through all the forms' named places and fixed the options in the named place form for all of them. They should be identical now. As a side note, the named places logic is an absolute nightmare right now: partially controlled in the named places form by a collectionID-switch, partially in separate forms that use JSON patches to extend the named place form... In the future, we should redesign the named place form data flow.

Reply:
> This was localized in some other place, should it be here also (https://211003.dev.laji.fi/en/about/5874)?

If we want to localize that, we'd have to do it in the app root router since the lang routing is done there. Since this is a problem only for direct links to the old route of the TOS, I'd much prefer not to add hacks for that.